optuna/integration/pytorch_lightning.py

```diff
@@ -40,6 +40,12 @@ class PyTorchLightningPruningCallback(Callback):
             ``pytorch_lightning.LightningModule.training_step`` or
             ``pytorch_lightning.LightningModule.validation_epoch_end`` and the names thus depend on
             how this dictionary is formatted.
+
+    .. note::
+        For the distributed data parallel training, ``accelerator="ddp"``,
+        the version of PyTorchLightning needs to be higher than or equal to v1.4.0.
+        In addition, :class:`~optuna.study.Study` should be instantiated with
+        :class:`~optuna.storages.RDBStorage`.
```

Comment:
Users can use `RDBStorage` without direct instantiation by specifying its URL, such as `study = optuna.create_study(storage="https://..."). How about not mentioning the concrete class name, and just saying "Study should be instantiated with RDB storage."?